src/components/NavBar/NavBarLink.vue

```diff
@@ -57,8 +57,15 @@
 		},
 		methods: {
 			closeNav() {
-				EventBus.$emit('navbar-closeVisibile');
-				EventBus.$emit('overlay-closeVisibile');
+				
+				var p = window.innerWidth / window.innerHeight;
+				if(p >= 1){
+					EventBus.$emit('overlay-closeVisibile');
+				} else {
+					EventBus.$emit('navbar-closeVisibile');
+					EventBus.$emit('overlay-closeVisibile');
+				}
+				
```

Comment:
The change I see here is to determine if the nav bar should be closed or not. If you look at one of the previous comments, here we could use the same `alwaysVisible()` function to determine if it always will be visible or not.